io/teca_cf_reader.cxx

```diff
@@ -789,6 +815,25 @@ teca_metadata teca_cf_reader::get_output_metadata(
                     step_count.push_back(tmp->size());
                 }
             }
+            else if (!this->t_values.empty())
+            {
+                // if time axis is provided manually by the user
+                size_t n_t_vals = this->t_values.size();
+                if (n_t_vals != files.size())
+                {
+                    TECA_ERROR("Number of files choosen doesn't match the"
+                        "the number of time values provided")
+                    return teca_metadata();
+                }
+
+                p_teca_variant_array_impl<double> t = teca_variant_array_impl<double>::New(n_t_vals);
+                for (size_t i = 0; i < n_t_vals; ++i)
+                {
+                    t->set(i, this->t_values[i]);
+                    step_count.push_back(1);
+                }
```

Comment:
FYI, this loop could be replaced with (something like)
```
p_teca_variant_array_impl<double> t = teca_variant_array_impl<double>::New(this->t_values.data(), n_t_vals);
step_count.resize(n_t_vals, 1);
```
Which is potentially more efficient because we'd let the vector handle the copy and allocation simultaneously.